.github/workflows/project_sync.yaml

```diff
@@ -0,0 +1,46 @@
+name: Add PRs and Issues to the Project Board
+
+on:
+  schedule:
+    - cron: '0 0,6,12,18 * * *'
```

Comment:
Do we want a different schedule?

Reply:
I agree with every hour unless there are rate limits.

And also, if possible a manual trigger.

Reply:
Rate limits are per hour anyways but are also fairly generous with the Github App setup, should be fine.

Reply:
Also will add manual triggering.